wca-regulations.md

```diff
@@ -173,6 +173,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 4f) Competition scramble sequences must be generated using a current official version of an official WCA scramble program (available [via the WCA website](https://www.worldcubeassociation.org/regulations/scrambles/)).
 - 4g) After scrambling a puzzle, the scrambler must verify that the puzzle is scrambled correctly. If the puzzle state is wrong, the scrambler must correct it (e.g. by solving the puzzle and applying the scramble sequence again).
     - 4g1) Exception: For the 6x6x6 Cube, 7x7x7 Cube, and Megaminx, it is not necessary to correct the puzzle state, at the discretion of the WCA Delegate.
+    - 4g2) Exception: For Clock, if the state of the pins are affected during transportation or uncovering, it is not necessary to correct the pins, at the discretion of the WCA Delegate.
```

Comment:
How can pins be corrected after failed uncovering? What if pins were scrambled incorrectly by a scrambler, would this attempt be investigated by a Delegate? Maybe it's better to mention that if pins misscramble was found after the beginning of the attempt (e.g. during the inspection, after the solve, after the attempt), then the attempt should stay and should not be DNF'ed/extra should not be given?

Reply:
After all, all the solutions we're getting to are sounding arbitrary and very open to abuse. I guess in the end the best solution would be to press all pins down or up :/

Reply:
I think making all pins up or down at the end of a scramble is much less favoured, so it would be unfortunate to settle for it because wording the alternative is difficult.

I think part of the problem is a lack of foundational regs regarding misscrambles. For example there is no reg that says the competitor must receive the correct scramble (only that the scrambler must scramble correctly).